package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dhis2-tracker-populator",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
This PR contains a new feature so this should be a minor version instead of a patch version. I also prefer not to do version bumps or tag releases on a feature branch like this because it introduces potential merge conflicts and duplication if there are multiple active branches at the same time. In the case it's probably okay though.